### ISSUE TITLE
NPM Warnings

### ISSUE THREAD

**Issue by toby63** [FreeTubeApp/FreeTube#1672]
**Behavior of the program**
The compilation process of the [freetube package on AUR](https://aur.archlinux.org/packages/freetube/) shows Warnings.

**Expected behavior**
No Warnings.

**To Reproduce**
Please add all steps to reproduce the behavior:
1. Build the Aur package.

**Environment Information (please complete the following information):**
 - OS: Arch Linux
 - OS Version: Recent
 - FreeTube version: 0.14.0
 - Installation Method (If applicable): -
 - Primary API Used: -

**Log:**

```
npm WARN old lockfile
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
npm WARN old lockfile
npm WARN old lockfile This is a one-time fix-up, please be patient...
npm WARN old lockfile
npm WARN deprecated yt-trending-scraper@2.0.1: Moved to https://www.npmjs.com/package/@freetube/yt-trending-scraper
npm WARN deprecated yt-dash-manifest-generator@1.1.0: Module moved to https://www.npmjs.com/package/@freetube/yt-dash-manifest-generator
npm WARN deprecated tsml@1.0.1: no longer maintained
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated eslint-plugin-standard@5.0.0: standard 16.0.0 and eslint-config-standard 16.0.0 no longer require the eslint-plugin-standard package. You can remove it from your dependencies with 'npm rm eslint-plugin-standard'. More info here: https://github.com/standard/standard/issues/1316
npm WARN deprecated babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
npm WARN deprecated debug@4.2.0: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated core-js@2.6.12: core-js@<3.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Please, upgrade your dependencies to the actual version of core-js.
```

**Comment by PikachuEXE** [comment]
Are you referring to "old lockfile" or "deprecated"?
Of course there can always be "why not both" :P

**Comment by toby63** [comment]
Yep, I refer to everything.

**Comment by PikachuEXE** [comment]
The `old lockfile` might be fixed by a recent commit.
For `deprecated` I think it's due to some dependency of dependency.
Not impossible to fix it but hard to figure out which package(s) is/are causing it.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@PikachuEXE didn't ur latest PR fix this?

**Comment by PikachuEXE** [comment]
@efb4f5ff-1298-471a-8973-3d47447115dc 
I think it only fixes `old lockfile` not `deprecated`

**Comment by toby63** [comment]
@efb4f5ff-1298-471a-8973-3d47447115dc No, I just compiled with the most recent commit fa7fb33cda4348a7cd3ebd332b4ee5a813304588 and everything seems to be fine.

Only one minor notice occured:
`npm notice New minor version of npm available! 8.5.5 -> 8.10.0`
